EG23.tex

```diff
@@ -294,83 +354,118 @@ \subsubsection{Case 1: Greedy Separations}
-reprocessing facility (i.e. one Cyclus prototype). The fuel input preferences
-have been set to 3 for all irradiated UOX fuel, 2 for higher breeding ratio
-SFR spent fuel, and 3 for the lower breeding ratio SFR spent fuel.  Since a higher
-number indicates a higher preference, these facilities will prefer to consume and process
-irradiated UOX fuel and low breeding ratio SFR fuel before they consume/process 
-high breeding ratio SFR fuel.  The characteristics of each of the
-reprocessing facilities used in Case 2 are summarized in Table
+The only difference with the Case 1 scenario is the way in which reprocessing
+capacity is handled over time.  Modeling on-demand separations is approximated by
+deploying many small separations facilities according to plutonium requirements
+for SFR fuel fabrication.  These Pu requirements were determined from the Case 1
+results. While the calculated deployment schedule does not follow plutonium usage
+exactly, it is a reasonable aproximation.
+
+For this calculation, all fuels are reprocessed by only one type of reprocessing
+facility (i.e. one Cyclus prototype). The fuel input preferences have been set to
+3 for all irradiated UOX fuel, 2 for higher breeding ratio SFR spent fuel, and 3
+for the lower breeding ratio SFR spent fuel.  Since a higher number indicates a
+higher preference, these facilities will prefer to consume and process irradiated
+UOX fuel and low breeding ratio SFR fuel before they consume/process high breeding
+ratio SFR fuel.  The implemented characteristics of each of the reprocessing
+facilities used in Case 2 are summarized in Table
 \ref{tab:fuelfab_2}.
 
 \begin{table}[h!]
     \centering
     \begin{tabular}{ll}
     \hline
-    Separation Properties	&	All Fuel	\\
+    Separation Implementation  &  All Fuel   \\
+    \hline
+    Throughput [tHM/month]     &  60         \\
+    Feed buffer [tHM]          &  66         \\
+    Pu output buffer  [tHM]    &  6          \\
+    Pu separation efficiency   &  0.99       \\
+    Recycled U buffer [tHM]    &  Unlimited  \\
+    U separation efficiency    &  0.99       \\
+    Waste buffer [tHM]         &  Unlimited  \\ 
+                                             \\
+    \multicolumn{2}{c}{Feed commmodities}    \\
+%    \hline
     \hline
-    Throughput [tML/month]	&	60		\\
-    Feed buffer [tML]		&	66		\\
-    Pu output buffer  [tML]	&	6		\\
-    Pu separation efficiency	&	0.99		\\
-    Recycled U buffer [tML]	&	Unlimited	\\
-    U separation efficiency	&	0.99		\\
-    Waste buffer [tML]		&	Unlimited	\\
+    Commodity Sources         &  Preferences \\
+    LWR UOX A                 &  3           \\
+    LWR UOX B                 &  3           \\
+    SFR MOX A                 &  2           \\
+    SFR MOX B                 &  1           \\
     \hline
     \end{tabular}
-    \caption{Separation facilities core properties.}
+    \caption{Separation facilities implementation.}
     \label{tab:fuelfab_2}
 \end{table}
```

Comment:
Reformat this table to have it match the way the Separation is implemented in Cyclus